packages/core/tests/utils/CoreHelperUtil.test.ts

```diff
@@ -28,4 +29,32 @@ describe('CoreHelperUtil', () => {
       expect(CoreHelperUtil.isAddress(address, chain)).toBe(expected)
     }
   )
+
+  it('should return true when inside an iframe', () => {
```

Comment:
Can we actually E2E test the scenario on the affected browser?

Reply:
@chris13524 I don't think we should add E2E tests for everything if unit tests are sufficient. If we see issues/regressions further down the line we can add E2E tests.

Reply:
The only tests added test that a helper function can detect if it's inside an iframe or not. It doesn't test that the bug itself was fixed which is the test (unit or E2E) that should exist

Reply:
This was a customer request, not really a bug.
We're refactoring our E2E tests which is currently a mess because we added too many unnecessary fixtures/variants.

The requirements to setup E2E for this are too much atm. We'll see if we can get away with more extensive integration tests (by mocking a bunch of stuff in the scaffold), if this is possible considering what the flow is doing. 

If we see any issues down the line, we can consider E2E.

Reply:
@chris13524 Addressed in https://github.com/reown-com/appkit/pull/3422